py_bind/optv/image_processing.pyx

```diff
@@ -0,0 +1,61 @@
+from optv.parameters cimport ControlParams, control_par
+import numpy as np
+from twisted.cred import error
+cimport numpy as np
+
+def preprocess_image(np.ndarray[ndim=2, dtype=np.uint8_t] input,
+                   int filter_hp,
+                   ControlParams control,
+                   int lowpass_dim=0,
```

Comment:
Didn't we find that this default is an error, that it should be at least 1?
I think lowpass_dim=0 would make this an identity function.